DuckDuckGo/NetworkProtection/AppTargets/BothAppTargets/VPNURLEventHandler.swift

```diff
@@ -33,6 +33,10 @@ final class VPNURLEventHandler {
     ///
     func handle(_ url: URL) async {
         switch url {
+        case VPNAppLaunchCommand.manageExcludedApps.launchURL:
```

Comment:
When using the menu options in the VPN menu app, we open the main app passing a URL.  This code here handles the menu actions.